Breeze.UI/src/app/wallet/tumblebit/password-confirmation/password-confirmation.component.html

```diff
@@ -8,7 +8,15 @@
         <h5 class="modal-title text-uppercase" id="modalReceive" *ngIf="!startingTumble">Password Confirmation</h5>
         <form class="col-10 mt-4 offset-1" [formGroup]="walletPasswordForm" *ngIf="!startingTumble">
           <div class="form-group clearfix">
-            <p class="text-muted text-center"><em>Please enter your source wallet password.</em></p>
+            <p class="text-muted text-center">
+              <em>By starting the Breeze Privacy Protocol you are subjecting your entire balance {{balance | coinNotation}}BTC to be shuffled by the Breeze Privacy Protocol.</em>
+            </p>
+            <p class="text-muted text-center">
+              <em>Each transaction will be of {{denomination}}BTC and will incur a fee of {{fee}}BTC. Additional fee’s may be incurred.</em>
+            </p>
```

Comment:
The way the fee shows up in example implies you will pay more in fees than tumble! I think `fee` is a measurement in satoshi or bits while `denomination` is in coin units. correct fee's to fees

Reply:
I just copy-pasted text provided by Mo and Iain. I have added word "total" in front of the fee